org.adempiere.base/src/org/compiere/model/I_AD_UserPreference.java

```diff
@@ -177,6 +177,15 @@ public interface I_AD_UserPreference
 	/** Get Detailed Zoom Across	  */
 	public boolean isDetailedZoomAcross();
 
+    /** Column name IsShowTechnicalInfOnHelp */
+    public static final String COLUMNNAME_IsShowTechnicalInfOnHelp = "IsShowTechnicalInfOnHelp";
```

Comment:
this is just for entity type ? perhaps should just use "IsShowEntityType" instead ?